scheduler/scripts/run_with_events.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+# For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+import os
+import logging
+from datetime import datetime
+
+from lucupy.minimodel.constraints import CloudCover, ImageQuality, Conditions, WaterVapor
+from lucupy.minimodel.site import ALL_SITES
+from lucupy.minimodel.semester import SemesterHalf
+from lucupy.observatory.abstract import ObservatoryProperties
+from lucupy.observatory.gemini import GeminiProperties
+
+from definitions import ROOT_DIR
+from scheduler.core.builder.blueprint import CollectorBlueprint, OptimizerBlueprint
+from scheduler.core.builder.builder import ValidationBuilder
+from scheduler.core.components.collector import *
+from scheduler.core.eventsqueue.nightchanges import NightChanges
+from scheduler.core.output import print_collector_info, print_plans
+from scheduler.core.programprovider.ocs import read_ocs_zipfile, OcsProgramProvider
+from scheduler.core.statscalculator import StatCalculator
+from scheduler.core.eventsqueue import EventQueue, WeatherChange
+from scheduler.services import logger_factory
+
+
+if __name__ == '__main__':
+    logger = logger_factory.create_logger(__name__, logging.INFO)
+    ObservatoryProperties.set_properties(GeminiProperties)
+
+    # Read in a list of JSON data
+    programs = read_ocs_zipfile(os.path.join(ROOT_DIR, 'scheduler', 'data', '2018B_program_samples.zip'))
+
+    # Create the Collector and load the programs.
+    collector_blueprint = CollectorBlueprint(
+        ['SCIENCE', 'PROGCAL', 'PARTNERCAL'],
+        ['Q', 'LP', 'FT', 'DD'],
+        1.0
+    )
+    queue = EventQueue()
+    weather_change = WeatherChange(new_conditions=Conditions(iq=ImageQuality.IQ20,
+                                                             cc=CloudCover.CC50,
+                                                             sb=SkyBackground.SBANY,
+                                                             wv=WaterVapor.WVANY),
+                                   start=datetime(2018, 10, 1, 10),
+                                   reason='Worst image quality',
```

Comment:
This is backwards. `IQ20` is the best image quality, and `IQANY` is the worst image quality. (Anything can be scheduled in `IQ20`, but in `IQANY`, only things with `IQANY` can be scheduled.)

Reply:
Thanks, my lack of astronomy was showing

Reply:
Remember to change this. I noticed it in the output yesterday. 😄